src/test/kotlin/uk/gov/justice/hmpps/offenderevents/e2e/HMPPSDomainEventsIntTest.kt

```diff
@@ -441,12 +353,9 @@ class HMPPSDomainEventsIntTest : QueueListenerIntegrationTest() {
     @Test
     @DisplayName("will ignore a deleted or non-existent offender")
     fun willIgnoreNonExistentOffender() {
-      // Wait for messages to have been sent to the prisoneventqueue
-      await().until { getNumberOfMessagesCurrentlyOnPrisonEventTestQueue() == 2 }
-      // Wait for messages to have been consumed by JMS
-      await().until { getNumberOfMessagesCurrentlyOnPrisonEventQueue() == 0 }
-      // Check that no hmpps event messages were generated from them
-      assertThat(getNumberOfMessagesCurrentlyOnHMPPSEventTestQueue()).isEqualTo(0)
+      await().untilAsserted {
```

Comment:
we just need to assert nothing was sent via the Spy rather than looking at queues